mlflow/types/llm.py

```diff
@@ -361,7 +399,7 @@ class ChatParams(_BaseDataclass):
         presence_penalty: (float): An optional param of positive or negative value,
             positive values penalize new tokens based on whether they appear in the text so far,
             increasing the model's likelihood to talk about new topics.
-        metadata (Dict[str, str]): An optional param to provide arbitrary additional context
+        custom_inputs (Dict[str, str]): An optional param to provide arbitrary additional context
```

Comment:
will we change this to type Any once serena's work lands?

Reply:
I think this can be a follow-up work, `convert_dataclass_to_schema` needs to be updated to support `Any -> AnyType` conversion.